packages/ufetch-ubuntu/ufetch-ubuntu.pacscript

```diff
@@ -0,0 +1,12 @@
+maintainer=("Vladislav Prokopenko <vladislavprokopenko@disroot.org>")
+pkgname="ufetch-ubuntu"
+arch=('amd64')
+pkgver="0.4"
+source=("https://gitlab.com/jschx/ufetch/-/raw/main/ufetch-ubuntu")
+license=('MIT')
+external_connection=true
```

Comment:
I just want to make sure you know what this line does. Because it's not needed for this program lol.
```suggestion
```

Reply:
yeah, I know what it does. I just thought it's needed